app/services/record_participant_event.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class RecordParticipantEvent
```

Comment:
Do we have unit tests for this file? If not, could we?

Reply:
Integration tests only for this service. It doesn't really stand well in isolation without lots of supporting database structure.